### PR TITLE
Branch about us

### PR DESCRIPTION
**Pull request by teoks0199**

### CODE REVIEW

File: docs/team/teoks0199.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (Ruizhi2001) [inline docs/team/teoks0199.md:1]: Detailed portfolio

File: docs/AboutUs.md [hunk 2]
```diff
@@ -19,14 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Teo Kai Sheng
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/teoks0199.png" width="200px">
```
Comment (Ruizhi2001) [inline docs/AboutUs.md:24]: Nice photo that fits the requirements!

**Review by Ruizhi2001 — APPROVED** [review]
LGTM

File: docs/AboutUs.md (hunk 2 above)
Comment (samuelmui8) [inline docs/AboutUs.md:24]: Nice photo!

File: docs/AboutUs.md [hunk 3]
```diff
@@ -19,14 +19,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/teoks0199)]
+[[portfolio](team/teoks0199.md)]
 
-* Role: Team Lead
+* Role: Developer
```
Comment (samuelmui8) [inline docs/AboutUs.md:29]: Good that you defined your role

**Review by samuelmui8 — APPROVED** [review]
LGTM!

**Pull request merged**